index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!--this is header section-->
+    <header class="header-container">
+        <h2 class="welcome">Welcome</h2>
+        <a class="menu-btn"><img src="/assets/images/Union.png" alt="menu button"></a>
+    </header>
+    <!--end of header section-->
+
+    <!--the main section-->
+    <main class="main-section-container">
+        <h2 class="main-title main-section-title">Hey there. I'm</h2>
+        <h2 class="main-title main-section-title">Maen</h2>    
+        <h2 class="main-title main-section-subtitle">I'm a software developer</h2>
```

Comment:
Please ensure that the main title (e.g., "I'm Marianne ...") is a `h1` tag according to the requirement.

Reply:
ok , but could we have tow h1 element ,
if not, how we could make different color for the same element,
because each part of the text has a different color.

Reply:
@maen1980 , 

According to the requirement. You can follow this order:
- The main title (e.g., "I'm Marianne ...") is a` h1` tag
- The subtitle (if existing) uses a header tag `hx` (any h tags) (examples: <h1>,<h2>,<h3>, etc...)
- The paragraph (e.g., "I'm a software ...") is a p tag